packages/react-components/src/organisms/__tests__/WorkingGroupCard.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+
+import WorkingGroupCard from '../WorkingGroupCard';
+
+const props: ComponentProps<typeof WorkingGroupCard> = {
+  id: '42',
+  title: 'My Working Group',
+  externalLink: 'https://www.google.com',
+  externalLinkText: 'Working Group Folder',
+  description: 'My Working Group Description',
+  lastModifiedDate: '2020-01-1',
+};
+
+it('renders the working group card', () => {
+  const { getByText } = render(
+    <WorkingGroupCard
+      {...props}
+      id="42"
+      title="My Working Group"
+      description="test description"
+      externalLinkText="Link text"
+      lastModifiedDate="2020-01-01"
+    />,
+  );
+  expect(getByText('My Working Group')).toBeVisible();
+  expect(getByText('test description')).toBeVisible();
+  expect(getByText('Link text')).toBeVisible();
+  expect(getByText('2020-01-01')).toBeVisible();
```

Comment:
Here don't we want the date to be the same pattern of the other cards like this one ?
<img width="677" alt="Screenshot 2022-11-29 at 10 34 36" src="https://user-images.githubusercontent.com/16595804/204542673-503b04c5-9c07-4adb-b902-6ff550720c51.png">

With the text `Last updated: ` in front of the date :)

<img width="756" alt="Screenshot 2022-11-29 at 10 34 23" src="https://user-images.githubusercontent.com/16595804/204542619-54fe9db6-95f1-46f0-b638-ba2fd24b11af.png">

Reply:
This was a big miss on my part, thank you Gabi!